backend/kernelCI_app/models.py

```diff
@@ -78,6 +79,7 @@ class Tests(models.Model):
     id = models.TextField(primary_key=True)
     origin = models.TextField()
     environment_comment = models.TextField(blank=True, null=True)
+    environment_compatible = ArrayField(models.TextField(), blank=True, default=list)
```

Comment:
shouldn't this also have the `null=True? `

Reply:
Done